docs/CONFIGURATION.md

```diff
@@ -63,9 +63,9 @@ Flags:
 
   -r, --repoName string            Repository name. Will be displayed in LaunchDarkly. Case insensitive. Repo names must only contain letters, numbers, '.', '_' or '-'."
 
-  -T, --repoType string            The repo service provider. Used to correctly categorize repositories in the LaunchDarkly UI. Aceptable values: github|bitbucket|gitlab|custom. (default "custom")
+  -T, --repoType string            The repo service provider. Used to correctly categorize repositories in the LaunchDarkly UI. Aceptable values: bitbucket|custom|github|gitlab. (default "custom")
 
-  -u, --repoUrl string             The display url for the repository. If provided for a github or bitbucket repository, LaunchDarkly will attempt to automatically generate source code links.
+  -u, --repoUrl string             The display url for the repository. If provided for a bitbucket, github, or gitlab repository, LaunchDarkly will attempt to automatically generate source code links.
```

Comment:
It would be good to use correct capitalization for Bitbucket, GitHub, GitLab

Reply:
hm, i would almost rather it say this instead

https://github.com/launchdarkly/ld-find-code-refs/blob/15723036994b6eac14506754283eae9c4d4126cf/options/flags.go#L43-L45